.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 - Fixes #?
 
 #### :clipboard: Subtasks
-- [x] Add `CHANGELOG` entry
+- [x] Add `CHANGELOG` upgrade notes, if required
```

Comment:
I suggest to remove the _check_ by default from the template.

```suggestion
- [ ] Add `CHANGELOG` upgrade notes, if required
```

Reply:
Yess! I was tempted to do this very same change. I guess there's no reason why this was checked by default